frontend/www/js/omegaup/components/user/PreferencesEdit.vue

```diff
@@ -231,5 +249,20 @@ export default class UserPreferencesEdit extends Vue {
       localeChanged: this.locale != this.profile.locale,
     });
   }
+
+  showWarningAboutRank(ev: Event): void {
+    $(ev.target as HTMLElement).popover('show');
+  }
+
+  handlePrivateProfileCheckboxChange(): void {
+    const privateProfileButton = this.$refs
+      .privateProfileButton as HTMLButtonElement;
+
+    if (this.isPrivate) {
+      $(privateProfileButton).popover('show');
+    } else {
+      $(privateProfileButton).popover('hide');
+    }
+  }
```

Comment:
En lugar de usar $refs, vamos a utilizar los `v-model` de vue. En este caso sólo tenemos que indicar que se muestre el popover cuando el valor del checkbox de la cuenta privada está seleccionado:
```suggestion
    this.show = this.isPrivate;
```

y ya sería todo.

Reply:
Gracias!